rxv/rxv.py

```diff
@@ -124,6 +125,30 @@ def on(self, state):
     def off(self):
         return self.on(False)
 
+    def play(self):
+        self._send_action('Play')
+
+    def pause(self):
+        self._send_action('Pause')
+
+    def stop(self):
+        self._send_action('Stop')
+
+    def next(self):
+        self._send_action('Skip Fwd')
+
+    def previous(self):
+        self._send_action('Skip Rev')
+
+    def _send_action(self, action):
```

Comment:
Rename it to something more specific. `_playback_control` or sth :) `_send_action` is too  generic